### ISSUE TITLE
requestPermission() fails [Android]

### ISSUE THREAD

**Issue by LuisRodriguezLD** [notifee/react-native-notifee#60]
`notifee.requestPermission()` throws an error on Android. I understand this method is mostly for iOS but it would be cool to have it resolve automatically on Android (like [messaging().requestPermission](https://rnfirebase.io/messaging/usage#ios---requesting-permissions))

![Imgur](https://i.imgur.com/ZPvfMOa.png)

**Comment by Ehesp** [comment]
This should be resolving straight away: https://github.com/notifee/react-native-notifee/blob/master/src/NotifeeApiModule.ts#L275

**Comment by LuisRodriguezLD** [comment]
What if it resolves to `true` instead? This allows the following [code](https://notifee.app/react-native/docs/ios/permissions#requesting-permissions) to work on both platforms 

```javascript
import notifee from '@notifee/react-native';

async function requestUserPermission() {
  const settings = await notifee.requestPermission();

  if (settings) {
    console.log('Permission settings:', settings);
  }
}
```
otherwise, users would need to use Platform.OS

**Comment by thundercipher** [comment]
Can I try to fix this?

**Comment by Ehesp** [comment]
`patch-package` would be a way to quickly fix, but we'll get it sorted this week.

**Comment by Salakar** [comment]
This has been fixed in 0.6.2 - changelog still to do but theres only 1 breaking change, renaming usages of `Importance` to `AndroidImportance`
